src/main/kotlin/lotto/component/LottoFactory645.kt

```diff
@@ -0,0 +1,27 @@
+package lotto.component
+
+import lotto.domain.Lotto
+
+object LottoFactory645 : LottoFactory {
+
+    const val RANGE_START = 1
+    const val RANGE_END = 45
+
+    const val SIZE_NUMBERS = 6
+
+    override fun generate(quantity: Int): List<Lotto> {
+        val lottoList = mutableListOf<Lotto>()
+        repeat(quantity) {
+            lottoList.add(
+                Lotto(
+                    RandomIntGenerator.generateSet(
+                        rangeStart = RANGE_START,
+                        rangeEnd = RANGE_END,
+                        size = SIZE_NUMBERS
+                    )
+                )
+            )
+        }
```

Comment:
indent가 너무 깊어요. 그리고 repeat을 활용해보려는 건 좋으나 코드의 가독성이 낮아지는 것 같네요. 
외부스코프에 리스트를 생성 이후 반복문내에서 값을 추가하는 것은 안되는 것은 아니지만 map과 같은 함수를 이용해서 더 간결하게 가능할 것 같네요.  
```suggestion
       return (0 until quantity).map {Lotto(generator.generate(...))}
```